R/processDivRates.R

```diff
@@ -90,20 +90,22 @@ processDivRates <- function(speciation_time_log = "",
                             fossilization_rate_log = "",
                             burnin = 0.25,
                             probs = c(0.025, 0.975),
-                            summary = "median") {
+                            summary = "median",
+                            timeline = NULL
```

Comment:
Could you make this change also in the documentation of the function?

Reply:
I added the doc strings, forgot this